ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -91,8 +91,7 @@ import           Ouroboros.Consensus.Storage.ChainDB.API.Types.InvalidBlockPunis
 import qualified Ouroboros.Consensus.Storage.ChainDB.API.Types.InvalidBlockPunishment as InvalidBlockPunishment
 import           Ouroboros.Consensus.Storage.Common
 import           Ouroboros.Consensus.Storage.FS.API.Types (FsError)
-import           Ouroboros.Consensus.Storage.LedgerDB.InMemory (LedgerDB)
-import qualified Ouroboros.Consensus.Storage.LedgerDB.InMemory as LedgerDB
+import           Ouroboros.Consensus.Storage.LedgerDB hiding (streamAll)
```

Comment:
I'd rather keep the qualified import here. It's hard to know where symbols come from when HLS fails, which happens often on my machine :tm:

Reply:
Done